src/placeos-driver/storage.cr

```diff
@@ -118,20 +118,25 @@ class PlaceOS::Driver::Storage < Hash(String, String)
   end
 
   def self.get(key)
-    client = new_redis_client
-    client.get(key.to_s)
-  ensure
-    client.try &.close
+    new_redis_client.get(key.to_s)
```

Comment:
is the redis client MT safe?

Reply:
doesn't look like it, if we use `Redis::PooledClient` then it would be
although we'd need to use this with the clustered redis client too

Reply:
I assume this would be simple enough to implement for clustered redis if it's not already?